packages/moonstone/Scrollable/ScrollableNative.js

```diff
@@ -463,61 +463,44 @@ class ScrollableBaseNative extends Component {
 		}
 
 		const
-			{childRef, containerRef, scrollToAccumulatedTarget} = this.uiRef,
+			{childRef, containerRef} = this.uiRef,
 			bounds = this.uiRef.getScrollBounds(),
-			canScrollVertically = this.uiRef.canScrollVertically(bounds),
-			pageDistance = (isPageUp(keyCode) ? -1 : 1) * (canScrollVertically ? bounds.clientHeight : bounds.clientWidth) * paginationPageMultiplier,
 			spotItem = Spotlight.getCurrent();
```

Comment:
If we rename `spotItem` to `focusedItem`, we can just put the variable name when calling `scrollToNextPage` or `scrollToNextITem`.